Sources/XcodeExport/Resources/LabelStyle.swift.stencil

```diff
@@ -0,0 +1,72 @@
+{% include "header.stencil" %}
+import UIKit
+
+public struct LabelStyle {
+
+    enum TextCase {
+        case uppercased
+        case lowercased
+        case original
+    }
+
+    let font: UIFont
+    let fontMetrics: UIFontMetrics?
+    let lineHeight: CGFloat?
+    let tracking: CGFloat
+    let textCase: TextCase
+    
+    init(font: UIFont, fontMetrics: UIFontMetrics? = nil, lineHeight: CGFloat? = nil, tracking: CGFloat = 0, textCase: TextCase = .original) {
+        self.font = font
+        self.fontMetrics = fontMetrics
+        self.lineHeight = lineHeight
+        self.tracking = tracking
+        self.textCase = textCase
+    }
+    
+    public func attributes(
+        for alignment: NSTextAlignment = .left,
+        lineBreakMode: NSLineBreakMode = .byTruncatingTail
+    ) -> [NSAttributedString.Key: Any] {
+        
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = alignment
+        paragraphStyle.lineBreakMode = lineBreakMode
+        
+        var baselineOffset: CGFloat = .zero
+        
+        if let lineHeight = lineHeight {
+            let scaledLineHeight: CGFloat = fontMetrics?.scaledValue(for: lineHeight) ?? lineHeight
+            paragraphStyle.minimumLineHeight = scaledLineHeight
+            paragraphStyle.maximumLineHeight = scaledLineHeight
+            
+            baselineOffset = (scaledLineHeight - font.lineHeight) / 4.0
+        }
+        
+        return [
+            NSAttributedString.Key.paragraphStyle: paragraphStyle,
+            NSAttributedString.Key.kern: tracking,
```

Comment:
curios to know why `NSAttributedString.Key.kern` is used here rather than `NSAttributedString.Key.tracking`?

Reply:
'tracking' is only available in iOS 14.0 or newer